.github/workflows/sdk.yml

```diff
@@ -268,6 +268,61 @@ jobs:
         env:
           TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
           TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+  
+  python-sdk-mac:
```

Comment:
checkout matrix : https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix

Reply:
may further improve it later due to the complexity of current process